src/pages/login/login.js

```diff
@@ -49,6 +49,26 @@ const Login = () => {
       .matches(formRegExp.password, errorMessages[language].value.password)
       .required(' ')
   });
+  const auth = () => {
+    const _onInit = (auth2) => {
```

Comment:
These handlers don't have any sense, since the user won't see `console.log`

Reply:
Fixed